tmcprototype/CspSubarrayLeafNode/CspSubarrayLeafNode.xmi

```diff
@@ -80,7 +80,7 @@
     </commands>
     <commands name="StartScan" description="" execMethod="start_scan" displayLevel="OPERATOR" polledPeriod="0" isDynamic="false">
       <argin description="">
-        <type xsi:type="pogoDsl:StringType"/>
+        <type xsi:type="pogoDsl:StringArrayType"/>
```

Comment:
Why do we change to StringArrayType if the input argument is JSON string?

Reply:
I order to comply with SubarrayNode and CspSubarray it is changed.
On SubarrayNode input argument type is DevVarStringArray. And it is inherited from base classes. So, in future if any issue comes with DevVarStringArray, then it will first come on SubarrayNode itself. Hence, we need to update base classes.